app/src/main/java/ca/fuwafuwa/kaku/Windows/CaptureWindow.java

```diff
@@ -56,7 +57,13 @@ public boolean onTouch(MotionEvent e) {
     @Override
     public boolean onDoubleTap(MotionEvent e) {
         tessStarted = true;
-        mTessRunnable.runTess(new BoxParams(params.x, params.y + getStatusBarHeight(), params.width, params.height));
+
+        int[] viewPos = new int[2];
+        mWindowBox.getLocationOnScreen(viewPos);
+
+        // TODO: Replace the first 1 with R.drawable.border_translucent.StrokeWidth when I figure out how
+        int offset = KakuTools.dpToPx(this.context, 1)+1;
+        mTessRunnable.runTess(new BoxParams(viewPos[0]+offset, viewPos[1]+offset, params.width-(2*offset), params.height-(2*offset)));
```

Comment:
Hmm... battery life is a good point. 

Don't you think could also be hard and error prone to double-tap a small capture box though? What if we introduced a artificial delay (500-1000ms) before actually triggering the OCR?

Actually, I just tested this a bit more on my device. I definitely think this feature should be implemented. It looks like Android will recognize a quick drag + tap again as a `onDoubleTap()` event. Here are two screenshots captured that still highlights this problem:

![screen 980911245648287](https://cloud.githubusercontent.com/assets/878158/22074838/9d8c5142-dd5e-11e6-944d-4aed3a7463cf.png)
![screen 981047879411482](https://cloud.githubusercontent.com/assets/878158/22074844/a1725d92-dd5e-11e6-9baf-baa94153c140.png)

This functionality might even need to be enhanced, seeing we'd still pass the current check on the second image. Might want to check at least 2 corners before considering that capture box has settled into place.